app/controllers/admin_controller.rb

```diff
@@ -2,9 +2,13 @@ class AdminController < ApplicationController
   before_action :authorize
   def index
     if params[:status].nil? || params[:status] == "all"
-      @orders = Order.all
+      @orders = all_orders
     else
       @orders = Order.where("status = ?", params[:status])
     end
   end
+
+  def all_orders
+    Order.all
+  end
```

Comment:
I removed it now. This was part of an abandoned attempt to refactor. :mount_fuji: